pkg/virt-handler/vm.go

```diff
@@ -586,10 +586,12 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 						if err != nil {
 							return err
 						}
-						if iface.PodIP != existingInterfaceStatusByName[domainInterface.Alias.Name].IP {
+
+						if iface.PodIP != existingInterfaceStatusByName[domainInterface.Alias.Name].IP ||
+							!reflect.DeepEqual(iface.PodIPs, existingInterfaceStatusByName[domainInterface.Alias.Name].IPs) {
```

Comment:
lets say in the domain
IP = 1.1.1.1
IPs = 1.1.1.1, fd10::0

and in the iface cache it equal
IP = 1.1.1.1
IPs = 1.1.1.1, fd10::1

as we let iface win in this case if IP was just different (as it was before this PR), we also let it win in case the IPs list is different

Reply:
In the scenario provided, the inequality is from the list, so just kipping the list comparison without the single one there is enough.If the IP field is different, it will be reflected in the list as well.
Do I miss something?

Reply:
nope done